src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/PublicTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator\Traits;
+
+/**
+ * @method $this public()
+ * @method $this private()
```

Comment:
missing `@author` on traits

Reply:
didn't want to pollute too much, so I decided to not add them on traits.

Reply:
Offtopic: there is no real point in `@author` tag today when we have VCS.

Reply:
there is to me: git blame reports CS fixers most of the time - and "author" is not "patcher".

Reply:
> there is to me: git blame reports CS fixers most of the time - and "author" is not "patcher".

`git log` ("history" in terms of GitHub)?

Reply:
I now I'm "author" of some classes that I did not write at all - the reason being that I drafted them in a discussion, and someone else implemented the feature we were talking about. So no, "code author" is not the same as "commit author".

Reply:
> So no, "code author" is not the same as "commit author".

I agree with it. But it still doesn't make `@author` tag useful for me, because it is very vague tag. If you really want to express your appreciation to someone, you can write in description as plain text and probably describe context of discussion, etc.

However, it turns out that most of the people use `@author` to mention commit author, they apply this tag even for very routine classes where there is no place for any kind of "intellectual property". 

Also this tag raises a lot of questions: what if original author doesn't want to be mention as an "author" of some piece of code? What if you copy-paste N% of code: does it mean that you should mention author of "original" code, even if copy-pasted code was very trivial? 

After all, when you notice bug in the class, you'd probably ask commit author in the first place.